duckduckgo/iOS
# Feature/compat content blocking

Reviewer: Mia
Asana: https://app.asana.com/0/414235014887631/427251562039376
CC:

**Description**:

Vertical slice of the content blocking work adding support for DisconnectMe and implementing the blocking logic define in:

https://app.asana.com/0/inbox/361428290920650/420243568919386/420318744704339

The Javascript layer logs excessively at the moment, I will tidy this up in the last PR of this work.

There's also a bug with detecting first party URLs - which I've added a comment about to the above Asana task.

There's also a hard dependency on webkit for messaging back to the app - I'd like to make this more generic so will add a layer of abstraction for that in a future PR.

**Steps to test this PR**:
1. Browse to a site with tracking
2. Open the javascript console and observe the logging. When it blocks it logs as "info"
3. Confirm blocked resources have not been loaded


###### Reviewer Checklist:


###### PR DRI Checklist:
